examples/scenes/src/cosmic_text_scene.rs

```diff
@@ -0,0 +1,425 @@
+// Copyright 2024 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
```

Comment:
A tiny bit of documentation would be good:
```suggestion

//! An example integrating Cosmic text with Vello.
//! This renders a read-only snapshot of a simplified text editor using system fonts.

```

Reply:
Done